### PR TITLE
Add new manageiq-providers-autosde repo

### PR DESCRIPTION
**Pull request by Fryguy**
@chessbyte Please review.

### CODE REVIEW

File: bin/new_plugin_repo.rb [hunk 1]
```diff
@@ -10,7 +10,7 @@
   ManageIQ::Release.common_options(self, :except => :repo_set)
 end
 
-repo = ManageIQ::Release.repo_for(opts[:repo])
```
Comment (chessbyte) [inline bin/new_plugin_repo.rb:13]: what is the rationale for this change?  seems unrelated to the PR title.

File: bin/new_plugin_repo.rb [hunk 2]
```diff
@@ -25,7 +25,7 @@
 ManageIQ::Release::UpdateLabels.new(repo.github_repo, opts).run
 
 puts "\n** Preparing Pull Request"
-ManageIQ::Release::PullRequestBlasterOuter.new(repo, opts).merge(
```
Comment (chessbyte) [inline bin/new_plugin_repo.rb:28]: what is the rationale for this change?  seems unrelated to the PR title.

File: bin/new_plugin_repo.rb (hunk 1 above)
Comment (Fryguy) [reply bin/new_plugin_repo.rb:13]: When I add new repos I run the bin/new_plugin_repo.rb, and these couple of changes were bugs found in that effort.
